plasmapy/diagnostics/langmuir.py

```diff
@@ -25,11 +25,21 @@
 from astropy.constants import si as const
 from astropy.visualization import quantity_support
 from scipy.optimize import curve_fit
+from warnings import warn
 
 from plasmapy.particles import Particle
 from plasmapy.utils.decorators import validate_quantities
 
 
+def _langmuir_futurewarning() -> None:
+    warn(
+        "The plasmapy.diagnostics.langmuir module will be deprecated in favor of "
+        "the plasmapy.analysis.swept_langmuir sub-package.  The analysis package "
+        " is in active development and is scheduled to be released with v0.5.0.",
```

Comment:
I think this warning message will need some updating.

Reply:
Done!